client/src/models/Person.js

```diff
@@ -34,6 +34,7 @@ export default class Person extends Model {
   static yupSchema = yup
     .object()
     .shape({
+      uuid: yup.string().default(""),
```

Comment:
Use `yup.string().nullable().default(null)` (a new object should have a null uuid, not the empty string).